robot/src/main/kotlin/org/team9432/robot/subsystems/amp/AmpIOReal.kt

```diff
@@ -1,20 +1,60 @@
 package org.team9432.robot.subsystems.amp
 
+import com.revrobotics.CANSparkBase
 import com.revrobotics.CANSparkLowLevel
 import com.revrobotics.CANSparkMax
+import com.revrobotics.SparkPIDController
 import edu.wpi.first.wpilibj2.command.SubsystemBase
 import org.team9432.robot.Devices
 
 class AmpIOReal: AmpIO, SubsystemBase() {
     private val spark = CANSparkMax(Devices.AMP_ID, CANSparkLowLevel.MotorType.kBrushless)
```

Comment:
If you want you can use the KSparkMAX thing here. It prevents it from calling the motor twice with the same setpoint and has some default parameters. You can leave it either way though